statsmodels/tsa/automatic/transform.py

```diff
@@ -0,0 +1,51 @@
+from scipy.stats.mstats import gmean
+import numpy as np
+import pandas as pd
+import statsmodels.api as sm
+from statsmodels import datasets
+
+
+def create_exog(nobs, d=0, seasonal_periods=1):
+    t = np.arange(1, nobs+1)
+    exog = []
+    for i in range(d+1):
+        exog.append(t**i)
+    if seasonal_periods > 1:
+        season = np.zeros((seasonal_periods-1, nobs))
+        for s in range(0, seasonal_periods):
+            for i in range(1, seasonal_periods):
+                for j in range(nobs):
+                    if j % seasonal_periods == i-1:
+                        season[i-1][j] = 1
+        for series in season:
+            exog.append(series)
+    return exog
+
+
+def predict_lambda(endog, lambda_range, nobs, d, seasonal_periods, offset=True):
```

Comment:
Can you add a citation to the Draper and Smith "Applied Regression Analysis" book that we followed in this implementation? Take a look at e.g. https://github.com/statsmodels/statsmodels/blob/master/statsmodels/regression/recursive_ls.py#L51 for how to format references in docstrings.

Reply:
What's the benefit of specifying `nobs` in the function definition? Shouldn't we just be getting `nobs = len(endog)`? Or am I missing something?

Reply:
Also, you should add a docstring.

Reply:
You're right. nobs=len(endog). I have kept that for testing. 
I'll add the docstring as well as the Reference as soon as possible and push the commit.